pkg/controller/controller.go

```diff
@@ -293,6 +293,28 @@ func Start(conf *config.Config, eventHandler handlers.Handler) {
 		go c.Run(stopCh)
 	}
 
+	if conf.Resource.ConfigMap {
+		informer := cache.NewSharedIndexInformer(
+			&cache.ListWatch{
+				ListFunc: func(options meta_v1.ListOptions) (runtime.Object, error) {
+					return kubeClient.CoreV1().ConfigMaps(meta_v1.NamespaceAll).List(options)
```

Comment:
i'm also thinking of make the namespace parameterized so that watch event on interested namespace only

Reply:
see #115